src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResource.kt

```diff
@@ -438,3 +513,20 @@ data class SentenceTermResponse(
   val endDate: LocalDate?,
   val lifeSentenceFlag: Boolean?,
 )
+
+@Schema(description = "Court case create request")
+data class CreateCourtCaseRequest(
+  @Schema(description = "Court case start date", required = true)
+  val startDate: LocalDate, // the warrant date in sentencing
+  @Schema(description = "Legal Case type", required = true)
+  val legalCaseType: String, // either A for Adult or sentences to create new type
+  @Schema(description = "Court Id (establishment)", required = true)
+  val court: String, // either A for Adult or sentences to create new type
+  @Schema(description = "Case status", required = true)
+  val status: String, // ACTIVE, INACTIVE, CLOSED
```

Comment:
will remove some of these code notes later, they are useful during analysis

Reply:
yes - that's right. I'll probably remove the Schema for these internal syscon ones if its not adding much